### PR TITLE
Release - API Management service: August, 2021

### PR DESCRIPTION
**Pull request by mikebudzynski**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -2,6 +2,38 @@
...
+5. API Management is now more responsive to moves and cluster scale-outs of Service Fabric backends.
+6. The `Network Status` management API response and page in the Azure portal now include details about the connectivity to Azure Application Insights for services with configured Application Insights loggers.
+7. The "Publish" button on the "Developer portal overview" page of API Management services in the Azure portal now uses the new `Portal revisions` API, which makes it easier to publish the portal for virtual-network-injected services.
+8. Clicking on a grid's row in the "Subscriptions" page of Azure API Management services in the Azure portal now brings up a new subscription editing interface.
+
+### Fixed
+
+1. We fixed an issue, which could cause schema definitions to be omitted while importing APIs from OpenAPI files.
+2. We fixed a paging issue for API calls for retrieving the list of APIs from the developer portal.
+3. The `PublisherEmail` service property now supports email addresses with the `#` character.
```
Comment (solankisamir) [inline changelogs/api-management-service.md:28]: We have added support for decrypting custom domain certificates encrypted with `AES256_SHA256` encryption algorithm.

Comment (solankisamir) [reply changelogs/api-management-service.md:28]: @jijohn14 this got enabled once you enabled Windows Server 2019 on RP.

**Review by solankisamir — CHANGES_REQUESTED** [review]
APIM Control Plane added support for decrypting certificates encrypted with `AES256_SHA256` algo

File: changelogs/api-management-service.md (hunk 1 above)
Comment (solankisamir) [reply changelogs/api-management-service.md:28]: https://github.com/MicrosoftDocs/azure-docs/issues/61561

Comment (mikebudzynski) [reply changelogs/api-management-service.md:29]: Added a bullet point, thanks!

**Review by solankisamir — APPROVED** [review]

**Pull request merged**